src/k8s-extension/azext_k8s_extension/_params.py

```diff
@@ -87,6 +87,10 @@ def load_arguments(self, _):
                    arg_group="Marketplace",
                    options_list=['--plan-publisher'],
                    help='The plan publisher is referring to the Publisher ID of the extension that is being taken from Marketplace portal under Usage Information + Support')
+        c.argument('skip_prerequisites',
```

Comment:
>skip_prerequisites [](http://example.com/codeflow?start=20&length=18)

Is this the standard across Az CLI commands to skip pre-reqs?
Or, is this something that is specific to Arc Autonomous and is being used for all the CLI commands for AA?

Reply:
It's intended to be an optional switch for k8s-extension only. For Arc Autonomous, this switch has to be specified anyway for installing Azure Monitor extensions. In my opinion, this would also be a useful flag for test purposes as I mentioned in description.

Reply:
Pasting from my other comment 'If we rely on a flag to be passed in by the user, if they don't, in the ArcAutonomous env., the pre-reqs will fail, and the experience wouldn't be good.
And, we can avoid this, if we detect and automatically do it without any input from the customer.'